frontend/src/components/tabs/RuntimeNodeDetailsV2.tsx

```diff
@@ -263,19 +263,14 @@ function ArtifactInfo({
 
   // Runtime artifact information.
   const createdAt = new Date(linkedArtifact.artifact.getCreateTimeSinceEpoch());
-  const lastUpdatedTime = linkedArtifact.artifact.getLastUpdateTimeSinceEpoch();
-  let finishedAt = '-';
-  if (lastUpdatedTime && linkedArtifact.artifact.getState() === Artifact.State.LIVE) {
```

Comment:
I feel that the origin of bug is here: the condition here should be `inkedArtifact.artifact.getState() !== Artifact.State.LIVE` instead of `inkedArtifact.artifact.getState() === Artifact.State.LIVE`.

Reply:
Thank you @zijianjoy for pointing out the problem, but I would like to ask if there's any scenario that artifact node will have or need `finishedAt` field.